ecs-cli/modules/compose/ecs/utils/convert_task_definition_test.go

```diff
@@ -28,12 +28,15 @@ import (
 )
 
 const (
-	portNumber    = 8000
-	portMapping   = "8000:8000"
-	containerPath = "/tmp/cache"
-	hostPath      = "./cache"
+	portNumber     = 8000
+	portMapping    = "8000:8000"
+	containerPath  = "/tmp/cache"
+	containerPath2 = "/tmp/cache2"
+	hostPath       = "./cache"
 )
 
+var emptyHostCtr = 0
```

Comment:
Will try again. Wherever I had put it before resulted in it overwriting itself every time it was called.